src/app/blocks/grid-block/grid-block.component.ts

```diff
@@ -60,7 +60,11 @@ export class GridBlockComponent implements OnInit, OnChanges {
     if (!!this.gridAngular && get(this.config, 'sizeColumnsToFit', true)) {
       setTimeout(() => {
         this.zone.run(() => {
-          this.gridAngular.api.sizeColumnsToFit();
+          try {
```

Comment:
I've attempted a number of approaches to this, but this is the only reliable one. 
I cannot seem to properly detect whether or not the api object exists. 

gridAngular exists. Api is an object and it's not null. But, when the sizeColumnsToFit runs, it's gone. 

try..catch seems to be the safest bet.

Reply:
I'd rather you put this comment in the code.